### DISCUSSION TITLE
Unable to use mesh_load_gmsh4 Matlab

### DISCUSSION THREAD

**Discussion by odysseas-96** [simnibs/simnibs#269] [6. Getting started]
Hey all,
matlab R2023b does not recognize the function.
Unrecognized function or variable 'mesh_load_gmsh4'.  Error in couple_gui (line 4) mesh = mesh_load_gmsh4
Have I to install any specific toolbox?
Teanks all

**Comment by Kstoffer** [comment]
Can you check that simnibs/matlab_tools is in your Matlab path (just write path in the Matlab command window) and if it is not add it. See for example: https://simnibs.github.io/simnibs/build/html/tutorial/scripting.html. If the error persists please write again.
Best,
Kristoffer
